src/components/transcript/index.tsx

```diff
@@ -24,7 +24,11 @@ import { useRouter } from "next/router";
 import { useEffect, useRef, useState } from "react";
 import MdEditor from "react-markdown-editor-lite";
 
-import type { TranscriptContent, UserReviewData } from "../../../types";
+import type {
+  SaveToGHData,
+  TranscriptContent,
+  UserReviewData,
+} from "../../../types";
```

Comment:
what kind of import is this?
those types shouldn't be on the root in the first place, therefore needing this kind of ugly import.
No need to do that in this PR, but at some point we should move them in the inside so that we can import them as the rest of the files.